telegram/dispatcher.py

```diff
@@ -33,7 +34,8 @@
 logging.getLogger(__name__).addHandler(H)
 
 semaphore = None
-running_async = 0
+async_threads = set()
+""":type: set[Thread]"""
```

Comment:
Does pycharm recognize `"""asyc_threads (set[Thread]): """`? This format would be preferrable, since it's a google style docstring.